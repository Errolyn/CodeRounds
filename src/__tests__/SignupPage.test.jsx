import SignupPage from '../signup/SignupPage';
import React from 'react';
import renderer from 'react-test-renderer';

it('renders the signup page correctly', () => {
    const tree = renderer
      .create(<SignupPage />)
      .toJSON();
    expect(tree).toMatchSnapshot();
});