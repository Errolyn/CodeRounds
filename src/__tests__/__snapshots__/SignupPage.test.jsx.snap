// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the signup page correctly 1`] = `
<div>
  Signup Page
</div>
`;
